src/main/java/me/totalfreedom/totalfreedommod/command/Command_deop.java

```diff
@@ -30,16 +31,17 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
                 player = onlinePlayer;
             }
         }
-
+        Player target = Bukkit.getPlayer(args[0]);
         // if the player is not online
         if (player == null)
         {
             player = DepreciationAggregator.getOfflinePlayer(server, args[0]);
         }
 
-        FUtil.adminAction(sender.getName(), "De-opping " + player.getName(), false);
+        FUtil.adminAction(sender.getName(), "De-opping " + player.getName(), true);
 
         player.setOp(false);
+        target.sendMessage(YOU_ARE_NOT_OP);
```

Comment:
You probably want to wrap this in an if statement otherwise you may send this message to someone who is already de-opped.